android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java

```diff
@@ -135,34 +136,25 @@ private Notification buildNotification() {
         final String projectName;
 
         nb = new NotificationCompat.Builder(this.context, "main-channel");
-        nb.setContentTitle(this.context.getResources().getQuantityString(
-                R.plurals.notice_notification,
+        nb.setContentTitle(this.context.getResources().getQuantityString(R.plurals.notice_notification,
                 notices = this.currentlyNotifiedNotices.size(),
-                projectName = this.currentlyNotifiedNotices.get(0).project_name,
-                notices)).
+                projectName = this.currentlyNotifiedNotices.get(0).project_name, notices)).
                 setSmallIcon(R.drawable.mailw).
                 setContentIntent(this.contentIntent);
         if (notices == 1) {
             // single notice view
             nb.setContentText(this.currentlyNotifiedNotices.get(0).title).
-                    setLargeIcon(NoticeNotification.getLargeProjectIcon(
-                            this.context,
-                            projectName)
-                    );
-        } else {
+                    setLargeIcon(NoticeNotification.getLargeProjectIcon(this.context, projectName));
+        }
+        else {
             // multi notice view
-            nb.setNumber(notices)
-                    .setLargeIcon(BitmapFactory.decodeResource(
-                            this.context.getResources(),
-                            R.drawable.ic_stat_notify_boinc_normal))
-                    .setSubText(this.context.getString(R.string.app_name));
+            nb.setNumber(notices).setLargeIcon(BitmapFactory.decodeResource(this.context.getResources(), R.drawable.ic_stat_notify_boinc_normal)).setSubText(this.context.getString(R.string.app_name));
```

Comment:
This is a type of change that makes things worse. The human editor had clear and justified intentions (with the line breaks) which should be preserved.